plugins/gfonts.koplugin/featured.lua

```diff
@@ -0,0 +1,61 @@
+local Featured = {
+    -- recommended fonts, both from gfonts and custom repos
```

Comment:
Recommended by you? :-)

Reply:
I took samples from https://www.mobileread.com/forums/showthread.php?t=316565 and the referenced issue. Some thread recommendations are not available on gfonts (tex gyrella, linux libertine...)